var/spack/repos/builtin/packages/py-griddataformats/package.py

```diff
@@ -15,12 +15,39 @@ class PyGriddataformats(PythonPackage):
     homepage = "http://www.mdanalysis.org/GridDataFormats"
     pypi = "GridDataFormats/GridDataFormats-0.5.0.tar.gz"
 
-    version("0.5.0", sha256="f317ed60708de22d1b2a76ce89a00f722d903291b1055ff1018d441870c39d69")
-    version("0.4.1", sha256="b362662c2dc475e2a3895fe044eaaa9a707bd660fd109a63dac84a47236690a3")
-    version("0.4.0", sha256="f81d6b75aa7ebd9e8b64e14558c2d2583a0589829382beb4ef69860110261512")
-    version("0.3.3", sha256="938f0efcb3bc2f58ec85048b933942da8a52c134170acc97cb095f09d3698fbd")
+    maintainers("RMeli")
+
+    version("1.0.1", sha256="ad2c9ab7d672a6d8c426de7d083eee4f3e2b0bd59391675d30683c768ab83cc4")
+    version(
+        "0.5.0",
+        sha256="f317ed60708de22d1b2a76ce89a00f722d903291b1055ff1018d441870c39d69",
+        deprecated=True,
+    )
+    version(
+        "0.4.1",
+        sha256="b362662c2dc475e2a3895fe044eaaa9a707bd660fd109a63dac84a47236690a3",
+        deprecated=True,
+    )
+    version(
+        "0.4.0",
+        sha256="f81d6b75aa7ebd9e8b64e14558c2d2583a0589829382beb4ef69860110261512",
+        deprecated=True,
+    )
+    version(
+        "0.3.3",
+        sha256="938f0efcb3bc2f58ec85048b933942da8a52c134170acc97cb095f09d3698fbd",
+        deprecated=True,
```

Comment:
What's the motivation for deprecation? (not that a good reason is required, just curious)

Reply:
Version `0.5` is already 4 years old, and still supports Python `2.7`. I'm not too keen to support such an old version (there have been other versions in between) since I added myself as a maintainer.

I can remove the deprecations if they are a bit too premature, but it's my way of saying I'm not actively looking into/testing the old versions.

Reply:
I'm fine with keeping the deprecations since you're maintaining the package now. But `2.7:` means 2.7+ which includes 3, so it's not like users _can't_ use it if they want to. But there are no packages that specifically require older versions of griddataformats, so it should be fine to deprecate them.

Reply:
> But there are no packages that specifically require older versions of griddataformats

Good point, thank you for checking!